content/gettingstarted/index.adoc

```diff
@@ -39,89 +39,66 @@ icon:lightbulb[size=1x]{nbsp}For several commands listed on this page, the OpenS
 
 icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
 
-To install the Kiali Operator (and, by default, the Kiali CR), run the install script via this command:
+During the installation, you will be prompted for the authentication strategy you want to use among `login`, `anonymous` and `openshift`.
+See the link:#_login_options[Login Options] section below for more information.
+
+=== Configure Kiali
+
+The Kiali Operator runs inside the `kiali-operator` namespace and listens for a Kiali CR (custom resource) creation or changes. When it sees one created, it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file].
+
+To edit from a terminal, run:
 
 [source,bash]
 ----
-bash <(curl -L https://git.io/getLatestKialiOperator)
+oc edit kiali kiali -n kiali-operator
 ----
 
-You will be prompted for the authentication strategy you want to use.
+Or to create / update using a local file (here `kiali.yml`):
 
-   * Choose `login` if you want to secure Kiali using a username and passphrase.
-   * Choose `anonymous` to remove authentication from Kiali and allow full access to Kiali without requiring any credentials.
-   * Choose `openshift` to use the OpenShift OAuth login which controls access based on the individual's RBAC roles in OpenShift (this is only a feature available when deployed on OpenShift - the Operator will fail to install Kiali successfully and will log an appropriate error message if you choose this when installing on Kubernetes).
-
-If the authentication strategy of "login" is selected, a secret containing Kiali login credentials is required to be deployed along with Kiali. In this case, the install script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The install script will store those credentials in a secret that is deployed in the same namespace where Kiali is installed.
-
-See the link:#_login_options[Login Options] section below for more information.
-
-icon:bullhorn[size=1x]{nbsp} If you told the install script to not create a Kiali CR (and thus not have Kiali installed yet) via the OPERATOR_INSTALL_KIALI=false environment variable, you are responsible for creating this secret if you wish to install Kiali with the authentication strategy of "login". A secret is not required if your authentication strategy is not "login". The following command is a simple way to create a secret for Kiali whose username is "admin" and passphrase is "admin":
 [source,bash]
 ----
-oc create secret generic kiali -n istio-system --from-literal "username=admin" --from-literal "passphrase=admin"
+oc apply -f kiali.yml -n kiali-operator
 ----
 
+=== Open the UI
```

Comment:
In this section, it doesn't tell the person how to log in (what credentials to use). To avoid frustration on the user's part, just put a quick blurb in here to tell them something like,

`The credentials you use on Kiali's login screen depend on the authentication strategy that was configured for Kiali. See "Login Options" for more details.`

Reply:
@jmazzitelli done